test/functional-portable/ucp/cloud/aws_test.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/radius-project/radius/pkg/ucp/api/v20231001preview"
 	"github.com/radius-project/radius/pkg/ucp/aws"
 	"github.com/radius-project/radius/pkg/ucp/frontend/controller/awsproxy"
+	ucptest "github.com/radius-project/radius/test/functional-portable/ucp"
```

Comment:
Is this the right home for this? 

If we add non-portable UCP tests are they going to reference `"github.com/radius-project/radius/test/functional-portable/ucp"` and is that ok?